OverlayPlugin.Core/NetworkProcessors/LineCountdown.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using RainbowMage.OverlayPlugin.MemoryProcessors;
+
+namespace RainbowMage.OverlayPlugin.NetworkProcessors
+{
+    [StructLayout(LayoutKind.Explicit, Size = structSize, Pack = 1)]
+    internal unsafe struct Countdown_v655
+    {
+        // 6.5.5 packet data (minus header):
+        // 34120010 4F00 1500 53  00  00  0102030405060708090A0B0C0D0E0F101112131415161718191A1B1C1D1E1F20 00000000
+        // AAAAAAAA CCCC BBBB DD  EE  FF  GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG HHHHHHHH
+        // 0x0      0x4  0x6  0x8 0x9 0xA 0xB
+        // Actor ID Wrld Time Res Unk Unk Name
+
+        public const int structSize = 48;
+        [FieldOffset(0x0)]
+        public uint actorID;
+        [FieldOffset(0x4)]
+        public ushort senderWorldId;
+
+        [FieldOffset(0x6)]
+        public ushort time;
+        [FieldOffset(0x8)]
+        public byte result;
+
+        [FieldOffset(0xB)]
+        public fixed byte senderName[32];
```

Comment:
More for my own edification than a requested change, but what was the thinking behind not including the unknown bytes at offset `0x09` and `0x0A`?  I know in the past we've included unknown fields as `param1`...`paramN`, so I'm curious about what drives that decision.

Reply:
I've never seen actual data in those fields, they've always been `00` bytes, so they might just be padding for alignment. If anyone has evidence that they can actually contain data, then I'm fine adding them.

Reply:
Makes sense (and I haven't seen any such evidence).  Although I suppose there's always some risk in omitting potential data (since adding it later would necessitate a new log line or a breaking change), in this case, I think the use cases for countdown data are small and largely known, and I can't imagine what other data about a countdown start that we'd ever need or use. Particularly since we have no idea if it actually is data vs. padding.

Reply:
The "register" log line for registering new custom lines includes a version number specifically to account for potentially breaking changes:

```
256|2024-02-25T18:24:15.6731077-05:00|268|OverlayPlugin|Countdown|1|4b4d718b647c232c
                                                                  ^ version number
```